environment.yml

```diff
@@ -27,7 +27,7 @@ dependencies:
   - bioconda::bedtools=2.29.2
   - conda-forge::libiconv=1.15
   - conda-forge::pigz=2.3.4
-  - bioconda::sequencetools=1.2.2
+  - bioconda::sequencetools=1.4.0.6
```

Comment:
That one will already come in via https://github.com/nf-core/eager/pull/449 but doesn't harm here either.